### ISSUE TITLE
The basic example using QML fails to work! 

### ISSUE THREAD

**Issue by Micrified** [therecipe/qt#1187]
**System**: macOS Catalina 10.15.7
**Go Version**: go1.15.4 darwin/amd64
---

Hello, 

I've been so excited to find this project on Github. Amazing work! I followed your instructions for installation on macOS, and was able to run your basic [widgets example](https://github.com/therecipe/examples/tree/master/basic/widgets). It looks great! 

Unfortunately, as soon as I try the [basic QML one](https://github.com/therecipe/examples/tree/master/basic/qml), I'm faced with the following error message: 

---
### Input
```qtdeploy test desktop github.com/therecipe/examples/basic/qml```

### Output

```
ERRO[0014] failed to run command
    _func=RunCmd cmd="go build -p 4 -v -ldflags=all=\"-w\" -trimpath -o /Users/micrified/Documents/Projects/Go/src/github.com/therecipe/examples/basic/qml/deploy/darwin/qml_project.app/Contents/MacOS/qml_project -tags=minimal"
    dir=/Users/micrified/Documents/Projects/Go/src/github.com/therecipe/examples/basic/qml
    env="QT_HOMEBREW=true GOARCH=amd64 CGO_ENABLED=1 XPC_SERVICE_NAME=0 TERM=xterm-256color USER=micrified GOFLAGS=-mod=vendor TMPDIR=/var/folders/dc/b3g27_h92n50677tl0cl1j240000gn/T/ ZSH=/Users/micrified/.oh-my-zsh _=/usr/local/bin/qtdeploy GOOS=darwin SHELL=/bin/zsh
    PWD=/Users/micrified/Documents/Projects/Go/src/aster
    TERM_PROGRAM_VERSION=433
    TERM_PROGRAM=Apple_Terminal
    SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.N1tLt3q48o/Listeners
    LSCOLORS=Gxfxcxdxbxegedabagacad
    HOME=/Users/micrified
    SHLVL=1
    LC_CTYPE=UTF-8
    PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/usr/local/MacGPG2/bin:/Library/Apple/usr/bin:/usr/local/go:/Users/micrified/Documents/Projects/Go/bin
    GOROOT=/usr/local/go
    XPC_FLAGS=0x0
    OLDPWD=/Users/micrified/Documents/Projects/Go/src
    PAGER=less
    GOPATH=/Users/micrified/Documents/Projects/Go
    TERM_SESSION_ID=CAAB2308-1D02-44D7-ADD3-89691117C89D
    LOGNAME=micrified LESS=-R"
    error="exit status 1"
    name="build for darwin on darwin"
../../vendor/github.com/therecipe/qt/gui/gui-minimal.go:13:2: cannot find package "." in:
	/Users/micrified/Documents/Projects/Go/src/github.com/therecipe/examples/vendor/github.com/therecipe/qt/internal/binding/runtime
```

In short, the file `gui-minimal.go` can't find package '.', which is supposed to be somewhere in your project files! I don't think I'm missing any installation steps. Have you any idea the cause?! 

Best, 

Micrified

**Comment by LuminousLizard** [comment]
Same issue for me on Linux Manjaro 5.8.18 and with Go 1.15.3 linux/amd64.

I tried to copy the "/github.com/therecipe/qt/internal/binding/runtime" from the library into the projects vendor folder. The error doesn't show again, but a another error raises for me: 

```
# github.com/therecipe/qt/internal/binding/runtime

moc.cpp:9:10: fatal error: QByteArray: File or directory not found
   9 | #include <QByteArray>
      |          ^~~~~~~~~~~~
Compiling terminated.
```
Compiling projects without qml works.

**Comment by ma6254** [comment]
```bat
C:\Users\mjc\git\goqt\00>qtdeploy build desktop github.com/ma6254/goqt/t_00\qml
time="2021-01-05T10:23:14+08:00" level=error msg="failed to run command" _func=RunCmd cmd="go build -p 12 -v -ldflags=all=\"-s\" \"-w\" \"-H=windowsgui\" -trimpath -o C:\\Users\\mjc\\git\\goqt\\00\\qml\\deploy\\windows\\qml_project.exe -tags=minimal" dir="C:\\Users\\mjc\\git\\goqt\\00\\qml" env="CDS_LIC_ONLY=1 ChocolateyLastPathUpdate=132503566570657870 DriverData=C:\\Windows\\System32\\Drivers\\DriverData HOMEPATH=\\Users\\mjc ALLUSERSPROFILE=C:\\ProgramData CDS_LIC_FILE=5280@LAPTOP-JCARMGUR ComSpec=C:\\WINDOWS\\system32\\cmd.exe PATH=C:\\Qt\\Qt5.13.0\\Tools\\mingw730_64\\bin;C:\\Qt\\Qt5.13.0\\5.13.0\\mingw73_64\\bin;C:/Qt/Qt5.13.0/Tools/mingw730_64\\bin;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Windows\\System32\\OpenSSH\\;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files\\NVIDIA Corporation\\NVIDIA NvDLISR;C:\\ProgramData\\chocolatey\\bin;C:\\Program Files\\Microsoft VS Code\\bin;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\WINDOWS\\System32\\OpenSSH\\;C:\\Program Files\\nodejs\\;C:\\Program Files\\dotnet\\;C:\\Program Files (x86)\\Altium Designer Summer 09\\System;C:\\Program Files\\Git\\cmd;C:\\Program Files (x86)\\Yarn\\bin\\;C:\\Program Files (x86)\\STMicroelectronics\\STM32 ST-LINK Utility\\ST-LINK Utility;C:\\ProgramData\\chocolatey\\lib\\openocd\\tools\\OpenOCD-20190828-0.10.0\\bin;C:\\Go\\bin;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\110\\Tools\\Binn\\;C:\\Program Files (x86)\\Microsoft SQL Server\\120\\Tools\\Binn\\;C:\\Program Files\\Microsoft SQL Server\\120\\Tools\\Binn\\;C:\\Program Files\\Microsoft SQL Server\\120\\DTS\\Binn\\;C:\\Program Files (x86)\\Windows Kits\\8.1\\Windows Performance Toolkit\\;C:\\Users\\mjc\\AppData\\Local\\Programs\\Python\\Python38\\Scripts\\;C:\\Users\\mjc\\AppData\\Local\\Programs\\Python\\Python38\\;C:\\Users\\mjc\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Users\\mjc\\go\\bin;C:\\Users\\mjc\\AppData\\Roaming\\npm;C:\\Users\\mjc\\AppData\\Local\\Pandoc\\;C:\\Program Files\\heroku\\bin;C:\\Program Files\\Bandizip\\;C:\\Users\\mjc\\AppData\\Local\\atom\\bin;C:\\Users\\mjc\\AppData\\Local\\Yarn\\bin;C:\\Users\\mjc\\AppData\\Local\\hyper\\app-3.0.2\\resources\\bin;C:\\Program Files\\STMicroelectronics\\STM32Cube\\STM32CubeProgrammer\\bin;C:\\Users\\mjc\\AppData\\Roaming\\Python\\Python38\\Scripts;C:\\Users\\mjc\\go\\bin ChocolateyInstall=C:\\ProgramData\\chocolatey COMPUTERNAME=LAPTOP-JCARMGUR KISYS3DMOD=C:\\Program Files\\KiCad\\share\\kicad\\modules\\packages3d PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 158 Stepping 10, GenuineIntel USERDOMAIN=LAPTOP-JCARMGUR BESIEGE_GAME_ASSEMBLIES=C:/Program Files (x86)/Steam/steamapps/common/Besiege/Besiege_Data\\Managed/ CHDL_LIB_INST_DIR=C:\\Cadence\\SPB_16.6 GOPATH=C:\\Users\\mjc\\go BESIEGE_UNITY_ASSEMBLIES=C:/Program Files (x86)/Steam/steamapps/common/Besiege/Besiege_Data\\Managed/ KICAD_SYMBOL_DIR=C:\\Program Files\\KiCad\\share\\kicad\\library OJDKBUILD_JAVA_HOME=C:\\Program Files\\ojdkbuild\\java-11-openjdk-11.0.6-1\\ OS=Windows_NT SW_SIM_MPIT=INTELMPI GOOS=windows GOARCH=amd64 ALTERA_FOR_ALTIUM_OVERRIDE=  CommonProgramFiles(x86)=C:\\Program Files (x86)\\Common Files CONCEPT_INST_DIR=C:\\Cadence\\SPB_16.6 PROCESSOR_LEVEL=6 PUBLIC=C:\\Users\\Public TMP=C:\\Users\\mjc\\AppData\\Local\\Temp ChocolateyToolsLocation=C:\\tools PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC ProgramFiles=C:\\Program Files PROMPT=$P$G CommonProgramW6432=C:\\Program Files\\Common Files ProgramData=C:\\ProgramData WT_SESSION=aba7bbba-f2e2-4a37-ad6a-44b151f30d97 PROCESSOR_REVISION=9e0a SystemDrive=C: GOFLAGS=-mod=vendor AltiumPath=C:\\Program Files (x86)\\Altium Designer Summer 09\\System APPDATA=C:\\Users\\mjc\\AppData\\Roaming PROCESSOR_ARCHITECTURE=AMD64 SW_SIM_HYDRA=C:\\Program Files\\Common Files\\SolidWorks Shared\\Simulation Worker Agent\\ windir=C:\\WINDOWS CGO_ENABLED=1 CommonProgramFiles=C:\\Program Files\\Common Files configsetroot=C:\\WINDOWS\\ConfigSetRoot HOMEDRIVE=C: PSModulePath=C:\\Program Files\\WindowsPowerShell\\Modules;C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules;C:\\Program Files (x86)\\Microsoft SQL Server\\120\\Tools\\PowerShell\\Modules\\ USERDOMAIN_ROAMINGPROFILE=LAPTOP-JCARMGUR TEMP=C:\\Users\\mjc\\AppData\\Local\\Temp =C:=C:\\Users\\mjc\\git\\goqt\\00 KISYSMOD=C:\\Program Files\\KiCad\\share\\kicad\\modules GOPROXY=https://goproxy.cn OneDrive=C:\\Users\\mjc\\OneDrive ProgramW6432=C:\\Program Files SW_SIM_TEMP=C:\\ProgramData\\SOLIDWORKS\\SW_net_sim_temp\\ GOROOT=c:\\go NUMBER_OF_PROCESSORS=12 SystemRoot=C:\\WINDOWS USERNAME=mjc ACTEL_FOR_ALTIUM_OVERRIDE=  KICAD_TEMPLATE_DIR=C:\\Program Files\\KiCad\\share\\kicad\\template LOGONSERVER=\\\\LAPTOP-JCARMGUR ProgramFiles(x86)=C:\\Program Files (x86) WSLENV=WT_SESSION::WT_PROFILE_ID WT_PROFILE_ID={0caa0dad-35be-5f56-a8ff-afceeeaa6101} CDSROOT=C:\\Cadence\\SPB_16.6 GO111MODULE=on LOCALAPPDATA=C:\\Users\\mjc\\AppData\\Local OA_PLUGIN_PATH=C:\\Cadence\\SPB_16.6\\Share\\oaPlugIns USERPROFILE=C:\\Users\\mjc" error="exit status 1" name="build for windows on windows"
..\vendor\github.com\therecipe\qt\gui\gui-minimal.go:13:2: cannot find package "." in:
        C:\Users\mjc\git\goqt\00\vendor\github.com\therecipe\qt\internal\binding\runtime


C:\Users\mjc\git\goqt\00>
```

same question

after many tests, I found that the last available version is commit 494d2f8c1ebe65dba7aee361a044511d67147357

```bat
PS C:\Users\mjc\git\goqt\00> go mod vendor
go: downloading github.com/therecipe/qt v0.0.0-20200812201058-80c87e74d442
PS C:\Users\mjc\git\goqt\00> qtdeploy test desktop ./qml
PS C:\Users\mjc\git\goqt\00> go mod vendor
go: downloading github.com/therecipe/qt v0.0.0-20200815005552-589db8f837c7
PS C:\Users\mjc\git\goqt\00> qtdeploy test desktop ./qml
PS C:\Users\mjc\git\goqt\00> ^C
PS C:\Users\mjc\git\goqt\00> go mod vendor
go: downloading github.com/therecipe/qt v0.0.0-20200817163748-494d2f8c1ebe
PS C:\Users\mjc\git\goqt\00> qtdeploy test desktop ./qml
PS C:\Users\mjc\git\goqt\00> go mod vendor
go: downloading github.com/therecipe/qt v0.0.0-20200817172549-0506a9f95d84
PS C:\Users\mjc\git\goqt\00> qtdeploy test desktop ./qml
time="2021-01-05T11:26:38+08:00" level=error msg="failed to run command" _func=RunCmd cmd="go build -p 12 -v -ldflags=all=\"-s\" \"-w\" \"-H=windowsgui\" -trimpath -o C:\\Users\\mjc\\git\\goqt\\00\\qml\\deploy\\windows\\qml_project.exe -tags=minimal" dir="C:\\Users\\mjc\\git\\goqt\\00\\qml" env="PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.CPL SystemRoot=C:\\WINDOWS LANG=zh_CN.UTF-8 CommonProgramFiles=C:\\Program Files\\Common Files HOMEDRIVE=C: LOGONSERVER=\\\\LAPTOP-JCARMGUR NUMBER_OF_PROCESSORS=12 VSCODE_GIT_IPC_HANDLE=\\\\.\\pipe\\vscode-git-67bcefc731-sock BESIEGE_UNITY_ASSEMBLIES=C:/Program Files (x86)/Steam/steamapps/common/Besiege/Besiege_Data\\Managed/ ChocolateyLastPathUpdate=132503566570657870 ChocolateyToolsLocation=C:\\tools ProgramFiles=C:\\Program Files SW_SIM_TEMP=C:\\ProgramData\\SOLIDWORKS\\SW_net_sim_temp\\ GIT_ASKPASS=c:\\Program Files\\Microsoft VS Code\\resources\\app\\extensions\\git\\dist\\askpass.sh AltiumPath=C:\\Program Files (x86)\\Altium Designer Summer 09\\System PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 158 Stepping 10, GenuineIntel PROCESSOR_REVISION=9e0a ProgramData=C:\\ProgramData DriverData=C:\\Windows\\System32\\Drivers\\DriverData OS=Windows_NT USERPROFILE=C:\\Users\\mjc https_proxy=http://127.0.0.1:7890 PUBLIC=C:\\Users\\Public USERDOMAIN_ROAMINGPROFILE=LAPTOP-JCARMGUR GOFLAGS=-mod=vendor CHROME_CRASHPAD_PIPE_NAME=\\\\.\\pipe\\crashpad_112500_WXACZQORQYVRLBYJ configsetroot=C:\\WINDOWS\\ConfigSetRoot FPS_BROWSER_USER_PROFILE_STRING=Default KISYS3DMOD=C:\\Program Files\\KiCad\\share\\kicad\\modules\\packages3d PATH=C:\\Qt\\Qt5.13.0\\Tools\\mingw730_64\\bin;C:\\Qt\\Qt5.13.0\\5.13.0\\mingw73_64\\bin;C:/Qt/Qt5.13.0/Tools/mingw730_64\\bin;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Windows\\System32\\OpenSSH\\;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files\\NVIDIA Corporation\\NVIDIA NvDLISR;C:\\ProgramData\\chocolatey\\bin;C:\\Program Files\\Microsoft VS Code\\bin;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\WINDOWS\\System32\\OpenSSH\\;C:\\Program Files\\nodejs\\;C:\\Program Files\\dotnet\\;C:\\Program Files (x86)\\Altium Designer Summer 09\\System;C:\\Program Files\\Git\\cmd;C:\\Program Files (x86)\\Yarn\\bin\\;C:\\Program Files (x86)\\STMicroelectronics\\STM32 ST-LINK Utility\\ST-LINK Utility;C:\\ProgramData\\chocolatey\\lib\\openocd\\tools\\OpenOCD-20190828-0.10.0\\bin;C:\\Go\\bin;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\110\\Tools\\Binn\\;C:\\Program Files (x86)\\Microsoft SQL Server\\120\\Tools\\Binn\\;C:\\Program Files\\Microsoft SQL Server\\120\\Tools\\Binn\\;C:\\Program Files\\Microsoft SQL Server\\120\\DTS\\Binn\\;C:\\Program Files (x86)\\Windows Kits\\8.1\\Windows Performance Toolkit\\;C:\\Users\\mjc\\AppData\\Local\\Programs\\Python\\Python38\\Scripts\\;C:\\Users\\mjc\\AppData\\Local\\Programs\\Python\\Python38\\;C:\\Users\\mjc\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Users\\mjc\\go\\bin;C:\\Users\\mjc\\AppData\\Roaming\\npm;C:\\Users\\mjc\\AppData\\Local\\Pandoc\\;C:\\Program Files\\heroku\\bin;C:\\Program Files\\Bandizip\\;C:\\Users\\mjc\\AppData\\Local\\atom\\bin;C:\\Users\\mjc\\AppData\\Local\\Yarn\\bin;C:\\Users\\mjc\\AppData\\Local\\hyper\\app-3.0.2\\resources\\bin;C:\\Program Files\\STMicroelectronics\\STM32Cube\\STM32CubeProgrammer\\bin;C:\\Users\\mjc\\AppData\\Roaming\\Python\\Python38\\Scripts;C:\\Users\\mjc\\go\\bin TEMP=C:\\Users\\mjc\\AppData\\Local\\Temp ALLUSERSPROFILE=C:\\ProgramData HOMEPATH=\\Users\\mjc ProgramW6432=C:\\Program Files ComSpec=C:\\WINDOWS\\system32\\cmd.exe FPS_BROWSER_APP_PROFILE_STRING=Internet Explorer OneDrive=C:\\Users\\mjc\\OneDrive USERDOMAIN=LAPTOP-JCARMGUR GOPATH=C:\\Users\\mjc\\go http_proxy=http://127.0.0.1:7890 SW_SIM_HYDRA=C:\\Program Files\\Common Files\\SolidWorks Shared\\Simulation Worker Agent\\ PSModulePath=C:\\Users\\mjc\\Documents\\WindowsPowerShell\\Modules;C:\\Program Files\\WindowsPowerShell\\Modules;C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules;C:\\Program Files (x86)\\Microsoft SQL Server\\120\\Tools\\PowerShell\\Modules\\ TERM_PROGRAM=vscode APPDATA=C:\\Users\\mjc\\AppData\\Roaming CDS_LIC_ONLY=1 CommonProgramFiles(x86)=C:\\Program Files (x86)\\Common Files PROCESSOR_ARCHITECTURE=AMD64 SW_SIM_MPIT=INTELMPI ChocolateyInstall=C:\\ProgramData\\chocolatey LOCALAPPDATA=C:\\Users\\mjc\\AppData\\Local Path=C:\\Qt\\Qt5.13.0\\5.13.0\\mingw73_64\\bin;C:/Qt/Qt5.13.0/Tools/mingw730_64\\bin;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Windows\\System32\\OpenSSH\\;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files\\NVIDIA Corporation\\NVIDIA NvDLISR;C:\\ProgramData\\chocolatey\\bin;C:\\Program Files\\Microsoft VS Code\\bin;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\WINDOWS\\System32\\OpenSSH\\;C:\\Program Files\\nodejs\\;C:\\Program Files\\dotnet\\;C:\\Program Files (x86)\\Altium Designer Summer 09\\System;C:\\Program Files\\Git\\cmd;C:\\Program Files (x86)\\Yarn\\bin\\;C:\\Program Files (x86)\\STMicroelectronics\\STM32 ST-LINK Utility\\ST-LINK Utility;C:\\ProgramData\\chocolatey\\lib\\openocd\\tools\\OpenOCD-20190828-0.10.0\\bin;C:\\Go\\bin;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\110\\Tools\\Binn\\;C:\\Program Files (x86)\\Microsoft SQL Server\\120\\Tools\\Binn\\;C:\\Program Files\\Microsoft SQL Server\\120\\Tools\\Binn\\;C:\\Program Files\\Microsoft SQL Server\\120\\DTS\\Binn\\;C:\\Program Files (x86)\\Windows Kits\\8.1\\Windows Performance Toolkit\\;C:\\Users\\mjc\\AppData\\Local\\Programs\\Python\\Python38\\Scripts\\;C:\\Users\\mjc\\AppData\\Local\\Programs\\Python\\Python38\\;C:\\Users\\mjc\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Users\\mjc\\go\\bin;C:\\Users\\mjc\\AppData\\Roaming\\npm;C:\\Users\\mjc\\AppData\\Local\\Pandoc\\;C:\\Program Files\\heroku\\bin;C:\\Program Files\\Bandizip\\;C:\\Users\\mjc\\AppData\\Local\\atom\\bin;C:\\Users\\mjc\\AppData\\Local\\Yarn\\bin;C:\\Users\\mjc\\AppData\\Local\\hyper\\app-3.0.2\\resources\\bin;C:\\Program Files\\STMicroelectronics\\STM32Cube\\STM32CubeProgrammer\\bin;C:\\Users\\mjc\\AppData\\Roaming\\Python\\Python38\\Scripts;C:\\Users\\mjc\\go\\bin PROCESSOR_LEVEL=6 SystemDrive=C: windir=C:\\WINDOWS TERM_PROGRAM_VERSION=1.52.1 TMP=C:\\Users\\mjc\\AppData\\Local\\Temp ACTEL_FOR_ALTIUM_OVERRIDE=  BESIEGE_GAME_ASSEMBLIES=C:/Program Files (x86)/Steam/steamapps/common/Besiege/Besiege_Data\\Managed/ CHDL_LIB_INST_DIR=C:\\Cadence\\SPB_16.6 CONCEPT_INST_DIR=C:\\Cadence\\SPB_16.6 GO111MODULE=on OJDKBUILD_JAVA_HOME=C:\\Program Files\\ojdkbuild\\java-11-openjdk-11.0.6-1\\ SESSIONNAME=Console GOOS=windows GOARCH=amd64 CommonProgramW6432=C:\\Program Files\\Common Files COMPUTERNAME=LAPTOP-JCARMGUR USERNAME=mjc KICAD_TEMPLATE_DIR=C:\\Program Files\\KiCad\\share\\kicad\\template OA_PLUGIN_PATH=C:\\Cadence\\SPB_16.6\\Share\\oaPlugIns ORIGINAL_XDG_CURRENT_DESKTOP=undefined CGO_ENABLED=1 ALTERA_FOR_ALTIUM_OVERRIDE=  CDS_LIC_FILE=5280@LAPTOP-JCARMGUR KICAD_SYMBOL_DIR=C:\\Program Files\\KiCad\\share\\kicad\\library GOROOT=c:\\go KISYSMOD=C:\\Program Files\\KiCad\\share\\kicad\\modules COLORTERM=truecolor VSCODE_GIT_ASKPASS_NODE=C:\\Program Files\\Microsoft VS Code\\Code.exe CDSROOT=C:\\Cadence\\SPB_16.6 GOPROXY=https://goproxy.io ProgramFiles(x86)=C:\\Program Files (x86) VSCODE_GIT_ASKPASS_MAIN=c:\\Program Files\\Microsoft VS Code\\resources\\app\\extensions\\git\\dist\\askpass-main.js" error="exit status 1" name="build for windows on windows"
..\vendor\github.com\therecipe\qt\gui\gui-minimal.go:13:2: cannot find package "." in:
        C:\Users\mjc\git\goqt\00\vendor\github.com\therecipe\qt\internal\binding\runtime

PS C:\Users\mjc\git\goqt\00> 
```

**Comment by samueldaviddelacruz** [comment]
did anyone found a solution for this issue? having the same problem

**Comment by teutat3s** [comment]
If you use go modules, try
```
go get -d github.com/therecipe/qt@494d2f8c1ebe65dba7aee361a044511d67147357
```
this downgrades the dependency to the commit mentioned as working by @ma6254

**Comment by Hareis** [comment]
yes, i have same problem
